db/database.go

```diff
@@ -315,12 +315,11 @@ func NewDatabaseContext(dbName string, bucket base.Bucket, autoImport bool, opti
 			}
 
 			if strings.Contains(name, "_") {
-				return nil, fmt.Errorf("OpenID Connect provider names cannot contain underscore:%s", name)
+				return nil, base.RedactErrorf("OpenID Connect provider names cannot contain underscore:%s", base.UD(name))
 			}
 			provider.Name = name
 			if _, ok := context.OIDCProviders[provider.Issuer]; ok {
-				base.Warnf(base.KeyAll, "Multiple OIDC providers defined for issuer %v", base.UD(provider.Issuer))
-				return nil, fmt.Errorf("Multiple OIDC providers defined for issuer %v", provider.Issuer)
+				return nil, base.RedactErrorf("Multiple OIDC providers defined for issuer %v", base.UD(provider.Issuer))
```

Comment:
Did we intend to drop the warning log here? I think this is OK, so long as the error is handled and logged further up the chain.

Reply:
Yes, I did that on purpose since it's basically duplicated and I think it's the responsibility of callers higher up the stack to handle this.  

In this case, it ends up getting returned to the REST API caller:

```
		return nil, base.HTTPErrorf(http.StatusBadRequest, fmt.Sprintf("OpenID Connect not configured for database %v", h.db.Name))

```

I don't know if we need it duplicated in the logs .. or if we do, can we do this more generically and log all HTTP responses that have error codes?